### PR TITLE
changed PresetTextWrap to PresetTextWarp

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: data/schemas/schemas_openxmlformats_org_drawingml_2006_diagram.json [hunk 1]
```diff
@@ -5023,7 +5023,7 @@
         },
         {
           "QName": ":custScaleY",
-          "PropertyName": "HeightScale",
+          "PropertyName": "HightScale",
```
Comment (mikeebowen) [inline data/schemas/schemas_openxmlformats_org_drawingml_2006_diagram.json:5026]: Is this undoing the fix from #1444 ?

File: generated/DocumentFormat.OpenXml.Linq/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/Linq.NoNamespace.g.cs [hunk 2]
```diff
@@ -4542,7 +4542,7 @@ public static class NoNamespace
         /// <para>As an XML attribute, it:</para>
         /// <list type="bullet">
         /// <item><description>is contained in the following XML elements: <see cref="DGM.prSet" />.</description></item>
-        /// <item><description>corresponds to the following strongly-typed properties: PropertySet.HeightScale.</description></item>
+        /// <item><description>corresponds to the following strongly-typed properties: PropertySet.HightScale.</description></item>
```
Comment (mikeebowen) [inline generated/DocumentFormat.OpenXml.Linq/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/Linq.NoNamespace.g.cs:4545]: Same question about #1444

File: generated/DocumentFormat.OpenXml/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/schemas_openxmlformats_org_drawingml_2006_diagram.cs [hunk 3]
```diff
@@ -6910,7 +6910,7 @@ public PropertySet(string outerXml) : base(outerXml)
         /// <para>Height Scale</para>
         /// <para>Represents the following attribute in the schema: custScaleY</para>
         /// </summary>
-        public Int32Value? HeightScale
+        public Int32Value? HightScale
```
Comment (mikeebowen) [inline generated/DocumentFormat.OpenXml/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/schemas_openxmlformats_org_drawingml_2006_diagram.cs:6913]: conflict with fix for #1444

File: generated/DocumentFormat.OpenXml/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/schemas_openxmlformats_org_drawingml_2006_diagram.cs [hunk 4]
```diff
@@ -7020,7 +7020,7 @@ internal override void ConfigureMetadata(ElementMetadata.Builder builder)
                 .AddAttribute("custSzX", a => a.FixedWidthOverride)
                 .AddAttribute("custSzY", a => a.FixedHeightOverride)
                 .AddAttribute("custScaleX", a => a.WidthScale)
-                .AddAttribute("custScaleY", a => a.HeightScale)
+                .AddAttribute("custScaleY", a => a.HightScale)
```
Comment (mikeebowen) [inline generated/DocumentFormat.OpenXml/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/schemas_openxmlformats_org_drawingml_2006_diagram.cs:7023]: Conflict with #1444

**Review by mikeebowen — CHANGES_REQUESTED** [review]
I think this was generated without the fix for #1444, which is needed before merging.

**Comment by tomjebo** [comment]
> I think this was generated without the fix for https://github.com/dotnet/Open-XML-SDK/pull/1444, which is needed before merging.

yup, you're right. It's because 1444 didn't get merged on the backend. I'll take care of that and it will fix this when I remerge everything.

**Comment by tomjebo** [comment]
Ok, HeightScale fix is now included.

**Review by mikeebowen — APPROVED** [review]

**Review by twsouthwick — DISMISSED** [review]
Lgtm - is there a spell check or something we could run over these property names to get them fixed before v3?

**Comment by tomjebo** [comment]
> Lgtm - is there a spell check or something we could run over these property names to get them fixed before v3?

yeah, we discussed this and we should do it anyway, but the problem is that things like "warp" to "wrap" don't trigger spell checkers. I'll give it a scan anyway.

**Comment by tomjebo** [comment]
I did a spell check scan on the front end code (non-generated) and found mostly comment spelling issues with only a couple code issues.

**Review by twsouthwick — DISMISSED** [review]
LGTM - although there's a merge conflict

**Comment by twsouthwick** [comment]
@tomjebo you'll need move the branch to the dotnet fork instead of yours until you get the GH actions working

**Comment by tomjebo** [comment]
> @tomjebo you'll need move the branch to the dotnet fork instead of yours until you get the GH actions working

Ah, ok. I'll move this over to a new dotnet branch.

**Comment by twsouthwick** [comment]
@tomjebo can this be closed?

**Pull request closed without merging**